Flow.Launcher.Plugin/Flow.Launcher.Plugin.csproj

```diff
@@ -56,7 +57,7 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <None Include="README.md" />
+    <None Include="Readme.md" Pack="true" PackagePath="\"/>
     <None Include="FodyWeavers.xml" />
```

Comment:
what do these change means? Does Readme.md work in Github?

Reply:
That's added so that the Readme file is rendered in nuget.org page of the package, see for example our [dotnet-new template](https://www.nuget.org/packages/Flow.Launcher.Plugin.Template#readme-body-tab)